src/Microsoft.TestPlatform.CrossPlatEngine/Client/InProcessProxyexecutionManager.cs

```diff
@@ -146,4 +146,8 @@ private void InitializeExtensions(IEnumerable<string> sources)
         // it will use TestPluginCache of vstest.console
         _executionManager.Initialize(Enumerable.Empty<string>(), null);
     }
+
+    public void InitializeTestRun(TestRunCriteria testRunCriteria, IInternalTestRunEventsHandler eventHandler)
+    {
```

Comment:
Could you add a comment about why we are leaving this empty.

Reply:
Fixed.